velox/exec/tests/PlanNodeToStringTest.cpp

```diff
@@ -422,25 +421,36 @@ TEST_F(PlanNodeToStringTest, exchange) {
 }
 
 TEST_F(PlanNodeToStringTest, tableScan) {
-  RowTypePtr rowType{ROW(
-      {"discount", "quantity", "shipdate"}, {DOUBLE(), DOUBLE(), VARCHAR()})};
-  auto filters = SubfieldFiltersBuilder()
-                     .add("shipdate", between("1994-01-01", "1994-12-31"))
-                     .add("discount", betweenDouble(0.05, 0.07))
-                     .add("quantity", lessThanDouble(24.0))
-                     .build();
-  auto tableHandle = exec::test::HiveConnectorTestBase::makeTableHandle(
-      std::move(filters), nullptr, "lineitem");
-  auto plan =
-      PlanBuilder()
-          .tableScan(
-              rowType,
-              tableHandle,
-              exec::test::HiveConnectorTestBase::allRegularColumns(rowType))
-          .planNode();
-
-  ASSERT_EQ("-> TableScan\n", plan->toString());
-  ASSERT_EQ(
-      "-> TableScan[Table: lineitem, Filters: [(discount, DoubleRange: [0.050000, 0.070000] no nulls), (quantity, DoubleRange: (-inf, 24.000000) no nulls), (shipdate, BytesRange: [1994-01-01, 1994-12-31] no nulls)]]\n",
-      plan->toString(true, false));
+  RowTypePtr rowType{
+      ROW({"discount", "quantity", "shipdate", "comment"},
+          {DOUBLE(), DOUBLE(), VARCHAR(), VARCHAR()})};
+  {
+    auto plan = PlanBuilder()
+                    .tableScan(
+                        rowType,
+                        {"shipdate between '1994-01-01' and '1994-12-31'",
+                         "discount between 0.05 and 0.07",
+                         "quantity < 24.0::DOUBLE"},
+                        "comment NOT LIKE '%special%request%'")
+                    .planNode();
+
+    ASSERT_EQ("-> TableScan\n", plan->toString());
+    const char* output =
+        "-> TableScan[table: hive_table, "
+        "range filters: [(discount, DoubleRange: [0.050000, 0.070000] no nulls), "
```

Comment:
I wonder if it makes sense to add toSql() method to Filter in some follow-up PR.

Reply:
That might work as well.